containers/ddev-webserver/Dockerfile

```diff
@@ -1,6 +1,6 @@
 FROM bitnami/minideb:stretch
 
-ENV PHP_VERSIONS="php5.6 php7.0 php7.1 php7.2 php7.3"
```

Comment:
This looks unintentional

Reply:
Looks like a rebase failure on my Windows world :(